spec/graphql/subscriptions_spec.rb

```diff
@@ -410,6 +416,20 @@ def str
             end
           end
 
+          it "avoid subscription on resolver error" do
+            res = schema.execute(<<-GRAPHQL, context: { socket: "1" }, variables: { "id" => "100" })
+          subscription ($id: ID!){
+            failedEvent(id: $id) { str, int }
+          }
+            GRAPHQL
+
+            # assert_equal nil, res["data"] # TODO: data is not nil for `FromDefinitionInMemoryBackend`
```

Comment:
I'm a bit confused about it here.. why the data is not nil for this type only..? Not sure if you have any idea 😂 

(If you don't have any quick idea, I can probably spent some time & investigate in the next few days)

Reply:
Hmm, I don't have a quick idea yet, but do you know what the value is, _instead_ of `nil`?

Reply:
Yeap. For `ClassBasedInMemoryBackend`, it's `nil`; for `FromDefinitionInMemoryBackend`, it's `{"failedEvent"=>nil}`. For both case, they all have a valid `errors` field though